frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -184,6 +177,16 @@ const ManageHostsPage = ({
     return query;
   })();
 
+  const initialPage = (() => {
+    let page = 0;
+
+    if (queryParams && queryParams.page) {
+      page = queryParams.page as number;
+    }
+
+    return page;
+  })();
+
```

Comment:
`initialPage` follows patterns established on component such as `initialQuery` and `initialSortBy`